docker/opensuse-tumbleweed-amd64-openssl-3.0.x/Dockerfile

```diff
@@ -11,7 +11,7 @@ COPY --from=utils . ./
 
 # Install system dependencies for Node.js
 RUN zypper refresh \
-  && zypper update \
+  && zypper --non-interactive dup \
```

Comment:
for the context, what’s the reason behind using `dup` instead of `update`?

Reply:
I had a message on the terminal stating "OpenSuse must be updated using `zupper dup`. Use the global `--non-interactive` if you don't have access to an interactive terminal" (or smth very close to that)